### DISCUSSION TITLE
Tailwindcss doesn't work for django

### DISCUSSION THREAD

**Discussion by dheerajd5** [williamboman/nvim-lsp-installer#503] [Q&A] [answered]
I have successfully installed the tailwindcss lsp using nvim-lsp-installer, but it seems that it doesn't recognize htmldjango as a filetype and hence no auto complete on these types of files.

**Comment by williamboman** [comment, marked as answer]
Hello! It seems like `htmldjango` [is not a configured filetype for tailwind in lspconfig](https://github.com/neovim/nvim-lspconfig/blob/50c821e2bf2182f4577168ee3bf959010582e177/lua/lspconfig/server_configurations/tailwindcss.lua#L14-L65). I see there's `django-html` though.

You can always [override the default settings](https://github.com/williamboman/nvim-lsp-installer/wiki/Advanced-Configuration#overriding-the-default-lsp-server-options), but in this case I think this ought to be fixed upstream in lspconfig.

**Reply by dheerajd5** [reply]
Okay thanks!

**Reply by williamboman** [reply]
This has now been added as a filetype to the default tailwind config in lspconfig, so no need to override anything!

**Reply by dheerajd5** [reply]
Ty so much
